rules/ssrf.data

```diff
@@ -33,6 +36,18 @@ http://2852039166/
 # IPv6 base
 http://[::]:
 
+# localhost bypass
+http://localtest.me
+http://127.0.0.1.nip.io
+http://127.127.127.127
+http://127.0.1.3
+http://127.0.0.0
+http://0177.0.0.1/
+http://[0:0:0:0:0:ffff:127.0.0.1]
```

Comment:
I think we can add also this:
https://github.com/swisskyrepo/PayloadsAllTheThings/pull/645